admin-dev/themes/new-theme/scss/components/_grid.scss

```
@@ -93,6 +93,15 @@ table .column-filters td:last-child .grid-reset-button {
             display: inline-block;
             width: 60px;
           }
+          .md-checkbox {
+            &:not(:last-child) {
+              display: inline-block;;
+            }
```

```suggestion
            }
            
```

Add a space to increase readability (would not ask you if I had nothing else to ask you 😄  )